EIPS/eip-4938.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 4938
+title: "eth/67: Removal of GetNodeData"
+author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
```

Comment:
When possible, GitHub user names are weakly preferred as our EIP tooling works better with them.  This isn't a hard requirement or blocker, just a nice-to-have if available.